File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -1116,7 +937,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Offline conversion
 
-The Blink HTML converter supports converting HTML to PDF in offline mode. While converting HTML to PDF in offline mode, the converter does not access the resources from the internet. This may increase the performance in slow internet connection.
+The Blink HTML converter supports converting HTML to PDF in offline mode using [EnableOfflineMode](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_EnableOfflineMode) property. While converting HTML to PDF in offline mode, the converter does not access the resources from the internet. This may increase the performance in slow internet connection.
```

Comment:
[EnableOfflineMode]() property in [BlinkConverterSettings]() class

Reply:
Modified